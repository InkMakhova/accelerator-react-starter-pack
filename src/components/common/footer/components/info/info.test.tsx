import React from 'react';
import { render } from '@testing-library/react';
import Info from './info';

it('should render Info correctly', () => {
  const {container} = render(<Info />);
  expect(container).toMatchSnapshot();
});
