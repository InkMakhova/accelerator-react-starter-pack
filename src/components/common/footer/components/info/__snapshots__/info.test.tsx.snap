// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Info correctly 1`] = `
<div>
  <section
    class="footer__nav-section footer__nav-section--links"
  >
    <h2
      class="footer__nav-title"
    >
      Информация
    </h2>
    <ul
      class="footer__nav-list"
    >
      <li
        class="footer__nav-list-item"
      >
        <a
          class="link"
          href="#top"
        >
          Где купить?
        </a>
      </li>
      <li
        class="footer__nav-list-item"
      >
        <a
          class="link"
          href="#top"
        >
          Блог
        </a>
      </li>
      <li
        class="footer__nav-list-item"
      >
        <a
          class="link"
          href="#top"
        >
          Вопрос - ответ
        </a>
      </li>
      <li
        class="footer__nav-list-item"
      >
        <a
          class="link"
          href="#top"
        >
          Возврат
        </a>
      </li>
      <li
        class="footer__nav-list-item"
      >
        <a
          class="link"
          href="#top"
        >
          Сервис-центры
        </a>
      </li>
    </ul>
  </section>
</div>
`;
